geobuf.proto

```diff
@@ -1,56 +1,38 @@
-option optimize_for = LITE_RUNTIME;
+message Object {
```

Comment:
@mourner total bikeshedding: what about ```GeoObject``` to add just a tiny bit of context with respect to application domain...